motor-stubs/core.pyi

```diff
@@ -731,7 +731,7 @@ class AgnosticBaseCursor(AgnosticBase):
     def get_io_loop(self) -> _IO_Loop: ...
     async def next(self) -> _Document: ...
     def next_object(self) -> _Document: ...
-    async def to_list(self, length: int) -> typing.List[_Document]: ...
+    async def to_list(self, length: typing.Optional[int]) -> typing.List[_Document]: ...
```

Comment:
According to my observations, we do need to provide a value to that parameter or python will error out. However, it does accept none as a value.

Reply:
A better alternative would be to use `typing.Union[int, None]`